ktlint/src/main/kotlin/com/pinterest/ktlint/internal/IntellijIDEAIntegration.kt

```diff
@@ -78,13 +78,20 @@ object IntellijIDEAIntegration {
             if (indentSize == 4) "" else "-is$indentSize"
             }"
             val paths =
-                // macOS
+                // macOS for 2019.3.x and below
                 Glob.from("IntelliJIdea*", "IdeaIC*", "AndroidStudio*")
                     .iterate(
                         Paths.get(home, "Library", "Preferences"),
                         Glob.IterationOption.SKIP_CHILDREN,
                         Glob.IterationOption.DIRECTORY
                     ).asSequence() +
+                    // macOS for 2020.1.x and above
+                    Glob.from("IntelliJIdea*", "IdeaIC*", "AndroidStudio*")
```

Comment:
Why separate list of `Paths`? Imho better to update line #84 with `, "Application Support", "JetBrains"`

Reply:
Updating line 84 to 
```
Paths.get(home, "Library", "Preferences", "Application Support", "JetBrains")
```
would cause us to look for ```"IntelliJIdea*", "IdeaIC*", "AndroidStudio*"``` under ```~/Library/Preferences/Application Support/JetBrains``` which wouldn't be right for the old or new version. That is ```Path.get(first, vararg more)``` is pretty much just a joining of the arguments.

But if it is preferable we could probably handle it via the glob-patterns, like
```Glob.from("**/IntelliJIdea*", "**/IdeaIC*", "**/AndroidStudio*"```?

Reply:
Changing the glob-pattern to what I suggested was very slow, and had very low precision (found a lot of cache/log folders).

Reply:
I see, thanks for clarifying :+1: